pages/en/billing.mdx

```diff
@@ -2,75 +2,74 @@
+> This section is written assuming you have deposited GRT into your billing account on [Subgraph Studio](https://staging.thegraph.com/studio/billing/) and that you're on the Arbitrum network.
 
-   a) In order to complete step #4, you must switch from Ethereum to the Polygon network. You can do this using the Subgraph Studio.
+1. Go to the [Subgraph Studio Billing page](https://staging.thegraph.com/studio/billing/).
 
-If you used your API keys over the course of any given week, you will receive an invoice based on the query fees you have generated during this period. This invoice will be paid using GRT available in your balance. Query volume is evaluated by the API keys you own. Your balance will be updated after fees are withdrawn.
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect".
 
-### Understanding Invoices in Subgraph Studio
+3. Click the dropdown next to the 'Add GRT' button at the center of the page. Select withdraw GRT. A side panel will appear.
 
-When using The Graph Network, dapps and users pay for query fees in GRT, the protocol's native work-utility token. Maintaining a positive balance for an account is the user's or dapp's responsibility.
+4. Enter the amount of GRT you would like to withdraw.
 
-Here are ways to be notified about and/or check a query fee balance:
+5. Click 'Withdraw GRT' to withdraw the GRT from your account. Sign the associated transaction in your wallet. This will cost gas. The GRT will be sent to your Arbitrum wallet.
 
-- You can set up a billing notification that will be sent to an email address [here](https://thegraph.com/studio/settings/).
+6. Once the transaction is confirmed, you'll see the GRT withdrawn from your account in your Arbitrum wallet within an hour.
 
-- You can query the billing subgraph API directly [here](https://thegraph.com/hosted-service/subgraph/graphprotocol/billing)
+### Adding GRT using a multisig wallet
 
-Also, you can learn how to (re)fill your query fee balance by watching [this video](https://www.loom.com/share/a81de6ef11d64c62872ea210c58c6af5).
+1. Go to the [Subgraph Studio Billing page](https://staging.thegraph.com/studio/billing/).
 
-> IMPORTANT: If there are unpaid invoices for more than two weeks, Indexers will stop serving queries for all API Keys associated with the account. Indexers will automatically start serving queries again once the billing balance is paid in full and GRT is added to the account for future query fees.
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect". If you're using [Gnosis-Safe](), you'll be able to connect your multisig as well as your signing wallet. Then, sign the associated transaction. This will not cost any gas.
```

Comment:
[Gnosis Safe] needs a link, potentially https://gnosis-safe.io/.

"Then, sign the associated transaction" -> I would change this to "Then, sign the associated message"

Reply:
Dang, missed that one lol.

Got those in there now!